.run/bsh-ui.sh

```
@@ -0,0 +1,65 @@
+# Default: rebuild
+rebuild=true
+
+# Update to the latest version of the main branch
+if [ -d "../${REPO_NAME}" ]; then
+  printf  "${COLOUR}Updating the local repository... ${LINE_END}"
+  cd ../${REPO_NAME}
+  git reset --hard
+  git checkout main
+  output=(git pull)
+
+  if echo "$output" | grep -q "Already up to date."; then
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)